the-toad-tribune/src/layout/Layout.tsx

```diff
@@ -52,3 +80,15 @@ const MainLayoutStyles = styled.div`
   grid-template-rows: repeat(9, 1fr);
   position: fixed;
 `;
+
+const MoreInfoStyles = styled.div`
+  height: 100vh;
```

Comment:
Careful with using vh on height as it might skew the elements inside it when moving the window vertically.

Reply:
Sounds good. I will check that out.